src/components/Cards/Cards.spec.js

```diff
@@ -0,0 +1,39 @@
+import {render, screen} from '@testing-library/react';
+import Cards from './Cards';
+
+const exampleExercises = [
+  {
+    id: 'e2',
+    bookmarked: false,
+    category: 'Augenübung',
+    title: 'Gucken und fühlen',
+    pathname: '/',
+    description: 'Blick vom Bildschirm weg in die Ferne schweifen lassen',
+    duration: '20 Sekunden',
+  },
+  {
+    id: 'e3',
+    bookmarked: false,
+    category: 'Bewegung',
+    title: 'Kurze Auflockerung',
+    pathname: '/',
+    description: 'Hinstellen, ganzen Körper strecken und Gelenke kreisen',
+    duration: '2 Minuten',
+  },
+];
+
+describe('Cards', () => {
+  it('renders the Cards component', () => {
+    render(
+      <Cards
+        onBookmark={() => console.log('Bookmark was clicked.')}
+        exercises={exampleExercises}
+        filter={exampleExercises.category}
+      />
+    );
+
+    const cards = screen.getByTestId('cards');
+
+    expect(cards).toBeInTheDocument();
```

Comment:
If you'd like to test the `onBookmark` function (but in another test case, i.d. another `it`:

```suggestion
  const mockedOnBookmarks = jest.fn();
    render(
      <Cards
        onBookmark={mockedOnBookmark}
        exercises={exampleExercises}
        filter={exampleExercises.category}
      />
    );

    const cards = screen.getByTestId('cards');

    expect(mockedOnBookmarks).toHaveBeenCalled();
```